src/App.tsx

```diff
@@ -1,18 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const pageNumber: number[] = [];
+  const [itemPerPage, setItemPerPage] = useState(3);
+  const [currentPage, setCurrentPage] = useState(1);
+  const pageNow = (page: number) => setCurrentPage(page);
+  const lastIndex = currentPage * itemPerPage;
+  const firstIndex = lastIndex - itemPerPage;
+  const currentItems = items.slice(firstIndex, lastIndex);
+
+  for (let i = 1; i <= Math.ceil(items.length / itemPerPage); i += 1) {
+    pageNumber.push(i);
+  }
+
+  const makeSTep = (direction: string) => {
+    if (direction === 'back' && currentPage > 1) {
+      setCurrentPage(prevState => prevState - 1);
+    }
+
+    if (direction === 'forward' && currentPage < pageNumber.length) {
+      setCurrentPage(prevState => prevState + 1);
+    }
+  };
+
+  function handleSelect(event: React.ChangeEvent<HTMLSelectElement>) {
+    setItemPerPage(+event.currentTarget.value);
```

Comment:
use `Number` instead `+`
```suggestion
    setItemPerPage(Number(event.currentTarget.value));
```